packages/ffe-account-selector-react/src/subcomponents/input-field/InputField.js

```diff
@@ -89,17 +97,24 @@ class Input extends Component {
                 {showReset && (
                     <button
                         className="ffe-base-selector__reset-button"
-                        onMouseDown={e => {
-                            e.preventDefault();
-                            onReset();
-                        }}
+                        onMouseDown={this.onReset}
                         tabIndex={-1}
                         type="button"
-                        aria-label="Reset"
                     >
-                        <KryssIkon className="ffe-base-selector__reset-button-icon" />
+                        <KryssIkon className="ffe-base-selector__reset-button-icon"/>
                     </button>
                 )}
+                <button
+                    className="ffe-base-selector__expand-button"
+                    onMouseDown={this.onExpandOrCollapse}
```

Comment:
Ser at det har vært sånn fra gammelt av, men `onClick` er å foretrekke. `onMouseDown` reagerer også på høyreklikk og klikk med scroll-hjulet.

Reply:
Grunne til at det er brukt `mouseDown` er for å forhindre at inputfeltet mister fokus når man trykker på knappen (ved å prevente `mouseDown` eventet, dette funker ikke for `click` eventet).